src/analysis/retail/priest/holy/modules/Abilities.ts

```diff
@@ -74,7 +74,7 @@ class Abilities extends CoreAbilities {
         healSpellIds: [SPELLS.DIVINE_HYMN_HEAL.id],
       },
       {
-        spell: SPELLS.SYMBOL_OF_HOPE.id,
+        spell: SPELLS.SYMBOL_OF_HOPE_TALENT.id,
```

Comment:
```suggestion
        spell: TALENTS.SYMBOL_OF_HOPE_TALENT.id,
```
these should be references to `TALENTS`. the `SPELLS` inclusion of talents is deprecated and is going to be removed.

Reply:
Did this for every spell made into a talent for Dragonflight and replaced dependencies in all files